lib/berbix.js

```diff
@@ -159,35 +171,45 @@ Client.prototype._fetchTokens = async function (path, payload) {
   );
 };
 
+Client.prototype._createTransaction = async function (path, payload) {
+  var headers = {
+    Authorization:
+      "Basic " + new Buffer.from(`${this.apiSecret}:`).toString("base64"),
+    "User-Agent": "BerbixNode/" + SDK_VERSION,
+  };
+  var result = await this._request("POST", path, headers, payload);
+  var tokens = new Tokens(
+    result.refresh_token,
+    result.access_token,
+    result.client_token,
+    now() + result.expires_in,
+    result.transaction_id,
+    result
+  );
+  return new TransactionResponse(tokens, {
+    hosted_url: result["hosted_url"],
+  });
+};
+
 Client.prototype.createTransaction = function (opts) {
-  var payload = {};
-  if (opts.email != null) {
-    payload.email = opts.email;
-  }
-  if (opts.phone != null) {
-    payload.phone = opts.phone;
-  }
-  if (opts.customerUid != null) {
-    payload.customer_uid = "" + opts.customerUid;
-  }
-  if (opts.templateKey != null) {
-    payload.template_key = opts.templateKey;
+  var payload = optionsToPayloadObject(opts);
+  var txnPromise = this._createTransaction("/v0/transactions", payload);
+  async function respTokens() {
+    var txnRes = await txnPromise;
+    return txnRes.tokens;
   }
-  if (opts.hostedOptions != null) {
-    var hostedOptions = {};
-    if (opts.hostedOptions.completionEmail != null) {
-      hostedOptions.completion_email = opts.hostedOptions.completionEmail;
-    }
-    if (opts.hostedOptions.redirectUrl != null) {
-      hostedOptions.redirect_url = opts.hostedOptions.redirectUrl;
-    }
-    payload.hosted_options = hostedOptions;
-  }
-  return this._fetchTokens("/v0/transactions", payload);
+  return respTokens();
 };
 
-// createUser is deprecated, please use createTransaction
-Client.prototype.createUser = Client.prototype.createTransaction;
+Client.prototype.createHostedTransaction = function (opts) {
+  var payload = optionsToPayloadObject(opts);
+  var txnPromise = this._createTransaction("/v0/transactions", payload);
+  async function hostedResp() {
+    var txnRes = await txnPromise
+    return {tokens: txnRes.tokens, hosted_url: txnRes.hostedProperties["hosted_url"]}
```

Comment:
```suggestion
    return {tokens: txnRes.tokens, hostedUrl: txnRes.hostedProperties["hosted_url"]}
```

Nit: camelCase for consistency with other props in the SDK interface